trunk/SUAVE/Components/Energy/Converters/Motor.py

```diff
@@ -53,8 +53,8 @@ def __defaults__(self):
         self.speed_constant     = 0.0
         self.propeller_radius   = 0.0
         self.propeller_Cp       = 0.0
-        self.gear_ratio         = 0.0
-        self.gearbox_efficiency = 0.0
+        self.gear_ratio         = 1.0
```

Comment:
I realized that some things were NaNing. i.e. the sped constant in lines 101,153 and 198 in Motor.py if the user doesn't specify a gear ratio. Since this is a parameter that is optional, we should make it an ideal motor (for new users) and allow the user to change it from that value.